install.sh

```diff
@@ -21,10 +21,21 @@ usage()
 # Return value of the latest published release on GitHub, with no heading "v" (e.g., "0.7.0")
 get_latest_release()
 {
-     curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
-     grep '"tag_name":'        |                                       # Get tag line
-     sed -E 's/.*"([^"]+)".*/\1/' |                                    # Pluck JSON value
-     sed -E 's/^v//'                                                   # Remove heading "v" if present
+     # Check if curl is installed
+    if command -v curl > /dev/null 2>&1; then
+       curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
+       grep '"tag_name":'        |                                       # Get tag line
+       sed -E 's/.*"([^"]+)".*/\1/' |                                    # Pluck JSON value
+       sed -E 's/^v//'                                                   # Remove heading "v" if present
+    elif command -v wget > /dev/null 2>&1; then
+       wget -q -O- "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
+       grep '"tag_name":'        |                                       # Get tag line
+       sed -E 's/.*"([^"]+)".*/\1/' |                                    # Pluck JSON value
+       sed -E 's/^v//'                                                   # Remove heading "v" if present
+    else
```

Comment:
You can reuse the logic later in the script where we define the `FETCH` variable.

Reply:
Looks better now - made it a function